snapshot/snapshot.go

```diff
@@ -37,6 +37,16 @@ import (
 const (
 	targetSnapshotLabel = "containerd.io/snapshot.ref"
 	remoteLabel         = "containerd.io/snapshot/remote"
+
+	// remoteSnapshot is a key for log line, which indicates whether `Prepare`
+	// method successfully prepared targeting remote snapshot or not, as defined
+	// in the following:
+	// - "true"  : indicates the snapshot has been successfully prepared as a
+	//             remote snapshot
+	// - "false" : indicates the snapshot failed to be prepared as a remote
+	//             snapshot
+	// - null    : undetermined
+	remoteSnapshot = "containerd.io/snapshot/prepare/remote.snapshot"
```

Comment:
too long maybe

Reply:
I thought to avoid name conflict because the key will be global but, yes I think so :thinking: